### ISSUE TITLE
The kong-dashboard server only listens to "localhost", not to its assigned IP address

### ISSUE THREAD

**Issue by DonMartin76** [PGBI/kong-dashboard#10]
After getting kong-dashboard up and running and trying to access it, it will only answer requests directed to `localhost:8080` (or `127.0.0.1:8080`). You cannot access the dashboard from a different system, as the gulp server doesn't listen to other network interfaces.

This may be intentionally, or not, I don't know.

The workaround for me was to add a `host` setting when creating the `webserver` instance (in `gulpFile.js`):

```javascript
    gulp.src('public')
        .pipe(webserver({
            livereload: false,
            host: "0.0.0.0",
            directoryListing: false,
            open: true,
            port: 8080
        }));
```

Adding `host: "0.0.0.0"` makes the server listen to all network interfaces.

**Comment by PGBI** [comment]
v1.0.1 is not using gulp anymore for serving the application. Do you still have this problem with v1.0.1?

**Comment by PGBI** [comment]
Just tested. Closing as this problem was solved in v1.0.1.

**Comment by DonMartin76** [comment]
Having this issue with the Dockerized version now (which I just created using docker build...). Will come back if I find out what's wrong.
